src/appConfig.ts

```
@@ -0,0 +1,59 @@
+}
+
+export const isBundledConfigUsed = !!process.env.INLINED_APP_CONFIG
+
+if (isBundledConfigUsed) {
+  loadAppConfig(process.env.INLINED_APP_CONFIG as AppConfig ?? {})
+} else {
+  void window.fetch('config.json').then(async res => res.json()).then(c => c, (error) => {
+  // console.warn('Failed to load optional app config.json', error)
+  // return {}
+    setLoadingScreenStatus('Failed to load app config.json', true)
+  }).then((config: AppConfig | {}) => {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)